installer/pkg/cmd/gcp_broker.go

```diff
@@ -145,27 +144,25 @@ func getOrCreateVirtualBroker(projectID, brokerName, brokerTitle string) (*virtu
 		return nil, fmt.Errorf("failed to create broker client. You might want to run 'gcloud auth application-default login'")
 	}
 
-	registryURL := "https://serviceregistry.googleapis.com"
-
-	res, err := brokerClient.GetBroker(&adapter.GetBrokerParams{
+	registryURL := "https://servicebroker.googleapis.com"
+	res, err := brokerClient.CreateBroker(&adapter.CreateBrokerParams{
 		RegistryURL: registryURL,
 		Project:     projectID,
 		Name:        brokerName,
+		Title:       brokerTitle,
+		Catalogs:    []string{"projects/gcp-services/catalogs/gcp-catalog"},
 	})
 	if err != nil {
```

Comment:
For this method, you actually will get a standard HTTP status back, rather than hacky OSB ones.

Reply:
I know, but the broker-cli client doesn't relay that information (http status etc.) at this level currently. It might be already fixed in newer version of broker-cli, but want to check with Joice.

Reply:
No, whatever in this repo is our latest broker-cli code. We didn't make any changes after that. I can update it if needed.